resources/ctia-default.properties

```diff
@@ -60,6 +60,12 @@ ctia.store.es.default.port=9205
 #ctia.store.es.default.host=es-storage.int.iroh.site
 #ctia.store.es.default.port=443
 #ctia.store.es.default.protocol=https
+#ctia.store.es.default.auth.type=api-key
+#ctia.store.es.default.auth.params.id=ngkvLnYB4ZehGW1qU-Xz
+#ctia.store.es.default.auth.params.api-key=6HMnACPRSVWSMvZCf9VcGg
```

Comment:
for dev purpose I assume we will mainly use basic auth with the default password defined for the container.
```suggestion
```